cmake/Codegen.cmake

```diff
@@ -53,16 +53,18 @@ install(FILES ${CMAKE_BINARY_DIR}/caffe2/core/macros.h
 
 # ---[ ATen specific
 if (INTERN_BUILD_ATEN_OPS)
-  SET(OPT_FLAG "-O3 ")
   IF(MSVC)
-    SET(OPT_FLAG "/Ox /fp:strict ")
-  ENDIF()
-  SET(VCOMP_LIB "vcomp")
-
-  IF("${CMAKE_BUILD_TYPE}" MATCHES "Debug")
-    SET(OPT_FLAG " ")
-    SET(VCOMP_LIB "vcompd")
-  ENDIF()
+    SET(OPT_FLAG "/fp:strict ")
+    # debug 'd' is added using <CONFIG> where ${VCOMP_LIB} gets referenced when using MSVC
+    SET(VCOMP_LIB "vcomp")
+  ELSE(MSVC)
+    SET(OPT_FLAG "-O3 ")
+    SET(VCOMP_LIB "vcomp")
```

Comment:
You can remove this one.

Reply:
Since `VCOMP_LIB` is used only within function `torch_set_target_props` of `cmake/public/utils.cmake`, may we should move its creation there completely? 
I don't find any reference of this variable anywhere except within the `if(MSVC AND AT_MKL_MT)` block.

Reply:
Agreed.